### DISCUSSION TITLE
Support Mac & Linux

### DISCUSSION THREAD

**Discussion by amd64fox** [SpotX-Official/SpotX#188] [General]
![Без названия](https://user-images.githubusercontent.com/62529699/197084273-239ccfc5-1cd3-4745-a1ad-9ebe30c8a31f.png)

We are glad to inform you that SpotX now supports Mac and Linux.
It's still being tested, but you can already try it out, also feel free to file bug reports.

Mac - https://github.com/SpotX-CLI/SpotX-Mac
Linux - https://github.com/SpotX-CLI/SpotX-Linux

Huge thanks to @jetfir3 and @Nuzair46 for a job well done.

**Comment by linuxmobile** [comment]
Amazing new :3
